app/languages/definitions/Catalan.yml

```diff
@@ -0,0 +1,14 @@
+locale: ca-ES
+dictionaryFile: ca-ES-utf8.csv
```

Comment:
`ca-utf8.csv` should be fine. In Portuguese, it makes sense to add the country code, because the European and the Brazilian variants are way too different. So, unless you are aiming for very specific Spanish-only Catalan, the country code can go away.